examples/polymer/vulcanize.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# vulcanize elements.html \
+#     --inline-script --inline-css --strip-comments | \
+#     crisper -h elements.build.html -j elements.build.js
+
+polybuild --maximum-crush elements/elements.html
```

Comment:
Removed

We need to keep vulcanize for a couple of reasons:

1. We encourage developers to use this tool for performance as a best practice. Our canonical todo app should be doing the same.
1. Polymer is different than other frameworks b/c it requires polyfills in other browsers. This app isnt too large, but html imports not being native in most browsers means slower perf in those browsers (if you're not vulcanizing). The last thing we want is devs running the demo and coming up with the the misperception that "Polymer is slow" b/c of this hinderance.